runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1851,6 +1852,10 @@ class INTERPRETER_CLASS
 
 	VMINLINE bool startAddressIsCompiled(UDATA extra) { return J9_ARE_NO_BITS_SET(extra, J9_STARTPC_NOT_TRANSLATED); }
 	VMINLINE bool methodIsCompiled(J9Method *method) { return startAddressIsCompiled((UDATA)method->extra); }
+	VMINLINE bool methodIsCompiledTRJIT(J9Method *method) { return _vm->jitConfig && startAddressIsCompiled((UDATA)method->extra); }
+#if defined(J9VM_OPT_MICROJIT)
+	VMINLINE bool methodIsCompiledMJIT(J9Method *method) { return _vm->jitConfig && startAddressIsCompiled((UDATA)method->extra); }
```

Comment:
These two functions appear to be identical apart from their names.  Intentional?

Reply:
It seems like we have made that method redundant, seems like a hold over from the `extra2` implementation. I'll have it removed for the next version. Thank you for catching this 👍